diff -Nuar setup.d.old/20copyfiles setup.d/20copyfiles
--- setup.d.old/20copyfiles	2015-06-27 15:40:37.818835526 +0300
+++ setup.d/20copyfiles	2015-06-27 15:42:53.918834475 +0300
@@ -83,11 +83,28 @@
                 if echo "$file" | egrep -q '^(#|$)' ; then
                     continue
                 fi
-                if echo "$file" | grep -q '^/'; then
-                    copy_file "$file" "${CHROOT_PATH}$file"
-                else
-                    warn "Not copying file with relative path: $file"
-                fi
+
+                case "$file" in
+                    *\ * )
+                        # warn "bosluk var: $file"
+                        src_file=${file%%" "*}
+                        dest_file=${file#*" "}
+                        if echo "$src_file" | grep -q '^/'; then
+                            copy_file "$src_file" "${CHROOT_PATH}$dest_file"
+                        else
+                            warn "Not copying file with relative path: $src_file"
+                        fi
+                        ;;
+                    *)
+                        # warn "bosluk yok"
+                        if echo "$file" | grep -q '^/'; then
+                            copy_file "$file" "${CHROOT_PATH}$file"
+                        else
+                            warn "Not copying file with relative path: $file"
+                        fi
+                        ;;
+                esac
+
             done < "$SETUP_COPYFILES"
         else
             fatal "copyfiles file '$SETUP_COPYFILES' does not exist"
